### DISCUSSION TITLE
Hello, Vladmandic, I would like to contribute some code to you.

### DISCUSSION THREAD

**Discussion by Scholar01** [vladmandic/sdnext#475] [Ideas]
I previously developed a plugin named "mov2mov" for Auto, but due to the plugin's features, including UI modifications, I had to do some hook operations to complete it. I would like to add an "m2m" tab after "i2i." What do you think? Would you consider merging it?

In addition, I have developed a cross-platform GUI that currently supports Windows, Linux, and Mac. It can automatically configure the required environment and provide convenient plugin installation. If you need it, I can open source and contribute it.

**Comment by Scholar01** [comment]
In addition, I can provide related fixes for Apple M1.

**Comment by vladmandic** [comment]
re: M1 - yes, please :)
re: GUI - can you share the link?
re: mov2mov plugin - sounds like that is best as an extension (and yes, adding new tab is perfectly fine). if you need some code added to main repo to support/enable extension, that is fine.

**Comment by Scholar01** [comment]
M1, yes, I am working on it and I am expected to submit a Pull Request later. From what I can see, it seems that your "installed" function is not judging accurately..

Gui:It is currently working for AUTO1111, and the language is currently only in Chinese. If you are interested, I can send it to you to take a look. If we want to adapt it to our branch, it may require additional work, including i18n (internationalization).

mov2mov, I will work on this later, but first I will make sure it runs properly on Mac.

Currently, I am able to work on this full-time because I am very passionate about it! Could you please leave a contact method so I can join in?

**Reply by vladmandic** [reply]
you can contact me on discord: vlado#8083
